vnext/e2e/__tests__/TestInputTest.ts

```diff
@@ -0,0 +1,41 @@
+import { webDriver2, HomePage, TextInputTestPage, AnyPage, clickAndWaitForHomePage } from '../testlib/index';
+import { waitForPage } from 'selenium-appium'
+
+jest.setTimeout(30000);
+
+beforeAll(() => {
+  return webDriver2.start();
+});
+
+afterAll(() => {
+  return webDriver2.stop();
+})
+
+describe("TestInputTestPage", () => {
+  beforeAll(async () => {
+    const homePage = await clickAndWaitForHomePage(webDriver2);
+    await homePage.clickAndWaitForTextInputTestPage();
+  })
+
+  test('First', async () => {
+    const textInputPage = await waitForPage(TextInputTestPage, webDriver2);
+    const textInput = textInputPage.getTextInput();
+    await textInput.clear();
+    await textInput.sendKeys('ABC');
+    expect(await textInput.getText()).toBe('ABC');
+  });
```

Comment:
Does this model allow control over how many tests are run when you do an npm run test?  Is this something jest lets you configure?  This doesn't have to be solved now, but eventually we'll have hundreds of tests so we'll need a way to specify which tests to run.

Reply:
jest supports filtering tests by name etc.  I think what you are suggesting is already supported.